### PR TITLE
IMB patterns and tests

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by itkovian** [comment]
Odd that tests pass, since the output locally contains no result data.

File: tests/data/jube [hunk 1]
```diff
@@ -87,4 +87,24 @@ data = [
...
+        "raw": "<14>1 2016-08-01T11:48:30.071651+02:00 node2602 jube[89798]: JUBE jube_id=21235 jube_label=None time=2016-08-01T11:45:23 seconds=1470044723 user=vsc40075 host=node2602 module=IMB/4.0.1-intel-2015a input=mpi1 pbs_job_id=21235 pbs_nodes=2 pbs_ppn= exec_host=node2602.swalot.gent.vsc/0-19,node2603.swalot.gent.vsc/0-19 cputime=751 walltime=176 compute_walltime= benchmark=Reduce bytes=1048576 iter=40 t_min_us=3576.84 t_max_us=3693.70 t_avg_us=3651.12 rate=",
+        "expected": {
+            u'cputime': 751,
+            u'exec_host': u'node2602.swalot.gent.vsc/0-19,node2603.swalot.gent.vsc/0-19',
+            u'input': u'mpi1',
+            u'jube_id': u'21235',
+            u'jube_label': u'None',
+            u'module': u'IMB/4.0.1-intel-2015a',
+            u'pbs_job_id': u'21235',
+            u'pbs_nodes': u'2',
```
Comment (wpoely86) [inline tests/data/jube]: int?

Comment (itkovian) [inline tests/data/jube]: Nope, this is the exec_host.

Comment (wpoely86) [inline tests/data/jube]: what do you mean? `exec_host` is a couple of lines higher?

Comment (itkovian) [inline tests/data/jube]: ah wait. hmm

Comment (itkovian) [inline tests/data/jube]: Well, it's similar. Maybe it needs some love, but I am not going to throw that in here. for now it is correct.

**Pull request merged**